SpringProject/src/main/java/com/gradproject2019/userConference/controller/UserConferenceController.java

```diff
@@ -0,0 +1,30 @@
+package com.gradproject2019.userConference.controller;
+
+import com.gradproject2019.userConference.data.UserConferenceRequestDto;
+import com.gradproject2019.userConference.data.UserConferenceResponseDto;
+import com.gradproject2019.userConference.service.UserConferenceService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+
+import javax.validation.Valid;
+import java.util.UUID;
+
+@CrossOrigin(origins = "http://localhost:3000")
+@Controller
+public class UserConferenceController {
+
+    @Autowired
+    private UserConferenceService userConferenceService;
+
+    @PostMapping(path = "/user-conferences")
```

Comment:
Has this been approved by a senior helper?

Reply:
I am not exactly sure what you are referring to here? we discussed the Post mapping path with Harry

Reply:
Fair enough, just bear in mind Harry isn't the only helper you should be learning from.